github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1111,9 +1111,9 @@ presubmits:
     annotations:
       fork-per-release: "true"
       testgrid-dashboards: kubevirt-presubmits
-    always_run: false
+    always_run: true
     optional: true
-    skip_report: false
+    skip_report: true
```

Comment:
We might have a problem with this, when skip_report is set to true, podinfo.json file is not created, it is only created as part of the reports here https://github.com/kubernetes/test-infra/blob/f7e21a3c18f4f4bbc7ee170675ed53e4544a0632/prow/crier/reporters/gcs/kubernetes/reporter.go#L208. We have this file as required for one of our lenses https://github.com/kubevirt/project-infra/blob/master/github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml#L48-L51 and when it is missing the job is set to failed. 

Could we keep skip_report as false?

Reply:
If skip_report is true, this will report to GitHub, which then will make tide block the merges, no?

Reply:
Doesn't `optional` influence the blocking?

Reply:
> Doesn't `optional` influence the blocking?

I think so, having `optional: true` the lane will be considered as non-required.

Reply:
In my experience even when the lane was marked as optional and the check failed, tide refused to automatically merge sometimes. Whether that was a bug or how that came otherwise I didn't investigage. I'm fine with letting the lane report, I just didn't want to create friction here.